release/models/system/openconfig-system-logging.yang

```diff
@@ -473,6 +479,142 @@ module openconfig-system-logging {
     }
   }
 
+  grouping logging-file-config {
+    description
+      "Configuration data for logfile";
+
+    leaf filename-prefix {
+      type string {
+        length 0..255;
+      }
+      description
+        "filename prefix. It is expected that implementation may append to it timestamp, serial-number or other type of identifier";
```

Comment:
```suggestion
        "A name used for the file.  It is expected that an implementation may append timestamp, serial-number or other identifier to the filename.";
```

Reply:
Changes are to avoid self-referential definition and making the text complete sentences with punctuation.